components/molecules/sidebar/components/sidebar.js

```diff
@@ -101,15 +158,29 @@ const StyledLi = styled(ListItem)`
       }
     `}
   }
+   {
+    /** Need to set the active  */
+  }
+  &:active {
+    ${({ theme }) => css`
+      background-color: purple;
+      border-right: 2px solid purple;
+    `}
+  }
 `
 
 const StyledLink = styled.span`
-  color: ${({ theme }) => theme.SIDEBAR.linkColour};
+  color: ${({ theme, styles }) =>
+    styles?.linkColor ? styles.linkColor : theme.SIDEBAR.linkColour};
   display: block;
+  display: flex;
+  justify-content: ${({ styles, isCollapsed }) =>
+    styles?.justifyContent ? styles.justifyContent : isCollapsed ? 'center' : 'start'};
```

Comment:
@esuarezz Prop over styles please - the use of styles if for third party developers only not for us really.